src/features/DisplayControls/DrugChart/components/CalendarHeader.jsx

```diff
@@ -1,27 +1,35 @@
 import React from "react";
 import PropTypes from "prop-types";
 import "../styles/CalendarHeader.scss";
+import data from "../../../../utils/config.json";
 
 export default function CalendarHeader(props) {
   const { currentShiftArray } = props;
-  const hours = currentShiftArray;
+  const enable24hour = data.config.drugChart.enable24HourTime;
 
   return (
     <div className="calendar-header">
       <div style={{ display: "flex" }}>
-        {hours.map((hour) => {
+        {currentShiftArray.map((hour) => {
+          const formattedHour = enable24hour
+            ? hour.toLocaleString("en-US", {
+                hour: "2-digit",
+                minimumIntegerDigits: 2,
+              })
+            : (((hour + 11) % 12) + 1).toLocaleString("en-US", {
```

Comment:
can we not do it like:
const formattedHour = `${(hour%12).toLocaleString("en-US", { minimumIntegerDigits: 2 })}
              :00`

Reply:
done